plugins/mac-virtualcam/src/camera-extension/cmake/macos/Info.plist.in

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleName</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleVersion</key>
+    <string>${MACOSX_BUNDLE_BUNDLE_VERSION}</string>
+    <key>CFBundleShortVersionString</key>
+    <string>${MACOSX_BUNDLE_SHORT_VERSION_STRING}</string>
+    <key>CFBundleExecutable</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleInfoDictionaryVersion</key>
+    <string>6.0</string>
+    <key>CFBundlePackageType</key>
+    <string>SYSX</string>
+    <key>CFBundleSupportedPlatforms</key>
+    <array>
+        <string>MacOSX</string>
+    </array>
+    <key>LSMinimumSystemVersion</key>
+    <string>13.0</string>
+    <key>NSHumanReadableCopyright</key>
+    <string>(c) 2012-${CURRENT_YEAR} Hugh Bailey</string>
+    <key>CMIOExtension</key>
+    <dict>
+        <key>CMIOExtensionMachServiceName</key>
+        <string>$(TeamIdentifierPrefix)$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+    </dict>
+    <key>NSSystemExtensionUsageDescription</key>
+    <string>This Camera Extension enables virtual camera functionality in OBS Studio.</string>
+    <key>OBSCameraDeviceUUID</key>
+    <string>${VIRTUALCAM_DEVICE_UUID}</string>
+    <key>OBSCameraStreamUUID</key>
+    <string>${VIRTUALCAM_STREAM_UUID}</string>
+    <key>OBSCameraStreamSinkUUID</key>
+    <string>${VIRTUALCAM_SINK_UUID}</string>
```

Comment:
Personally not a great fan of this, can't fully put into words why. The Info.plist doesn't quite feel right for this kind of attribute. I had solved it by generating a C-Style header and using it as a bridgingheader, but I'm not sure that's optimal either.

Can you think of *some* other way? The answer might be "no" in the end, but maybe you can think of something else that I don't.

Reply:
The `Info.plist` is to my knowledge actually the canonical way to put such information - this is where you add which services your application provides, which file types you support, which extensions you hold, etc. pp.

In essence, any meta "information" about your application should go into that file and the Cocoa functions exist for that reason - it's an easy way to read such information (which is also why we store information for Sparkle in there).

The plist itself cannot be changed after bundling/codesigning without invalidating the signature.

Reply:
Ah, I see 👍